components/email/src/Email.svelte

```diff
@@ -959,21 +1003,42 @@
                   </div>
                 {/if}
                 <div class="from-participants">
-                  {#if activeThread.messages.length >= 1 && activeThread.messages[activeThread.messages.length - 1].from.length}
-                    <span class="from-sub-section"
-                      >{activeThread.messages[activeThread.messages.length - 1]
-                        .from[0].name ||
-                        activeThread.messages[activeThread.messages.length - 1]
-                          .from[0].email}</span
-                    >
-                  {/if}
-                  {#if activeThread.messages.length > 1 && activeThread.messages[0].from.length && activeThread.messages[0].from[0].email !== activeThread.messages[activeThread.messages.length - 1].from[0].email}
-                    ,
-                    <span class="from-sub-section"
-                      >{activeThread.messages[0].from[0].name ||
-                        activeThread.messages[0].from[0].email}</span
-                    >
-                  {/if}
+                  <div class="participants-name">
+                    {#if activeThread.messages.length >= 1 && activeThread.messages[activeThread.messages.length - 1].from.length}
+                      <span class="from-sub-section"
+                        >{activeThread.messages[
+                          activeThread.messages.length - 1
+                        ].from[0].name ||
+                          activeThread.messages[
+                            activeThread.messages.length - 1
+                          ].from[0].email}</span
+                      >
+                    {/if}
+                    {#if activeThread.messages.length > 1 && activeThread.participants.length >= 2 && activeThread.messages[0].from.length && activeThread.participants[0].email !== activeThread.messages[activeThread.messages.length - 1].from[0].email}
```

Comment:
Could you look into improving the readability of this code? The variable paths here are getting kind of crazy, and it's hard to understand what we're checking for here. Moving this into a function and creating some intermediate variables could help.